invokeai/app/api/routers/sessions.py

```diff
@@ -37,6 +38,37 @@ async def create_session(
     return session
 
 
+@session_router.post(
+    "/batch",
+    operation_id="create_batch",
+    responses={
+        200: {"model": BatchProcess},
+        400: {"description": "Invalid json"},
+    },
+)
+async def create_batch(
+    graph: Optional[Graph] = Body(default=None, description="The graph to initialize the session with"),
+    batches: list[Batch] = Body(description="Batch config to apply to the given graph"),
+) -> BatchProcess:
+    """Creates and starts a new new batch process"""
+    batch_id = ApiDependencies.invoker.services.batch_manager.create_batch_process(batches, graph)
+    ApiDependencies.invoker.services.batch_manager.run_batch_process(batch_id)
```

Comment:
Can we return a list of the sessionIds that were created/invoked here?

Reply:
I've gone back and forth about making the session creation async on its own thread. At about 500 sessions it takes roughly 5 seconds to create all the sessions and start running. I have it just returning the batch_id currently since they wouldn't be available to return if we made that async. What are your thoughts on that?